docs/schema/records_reference.md

```diff
@@ -28,81 +13,112 @@ The versioned release format is designed to support analysis of how data has cha
+
+A record **should** contain a [compiledRelease](#compiled-release) object, which represents the state of the contracting process at the time of the record's publication.
 
-* Releases (required)
-* Compiled Release (recommended)
-* Versioned Release (optional)
+A record **may** contain a [versionedRelease](#versioned-release) object, which aggregates all values from all releases in a single object. The versioned release is designed to make it easy to see how values change from one release to another, and will often be generated by data users, rather than by publishers.
 
-### Release list
+### Releases
 
-The releases that make up a contracting process can be provided in two ways:
+Each release in a record can be provided as either a linked release or an embedded release.
 
-* URLs and metadata for each release; or
-* Embedded copies of the release;
+#### Linked releases
+
+A linked release follows a simple schema:
+
+```eval_rst
+.. jsonschema:: ../../build/current_lang/record-package-schema.json
+   :pointer: /definitions/record/properties/releases/oneOf/0
+```
 
-If providing an array of URLs, it must be possible for a consuming application to look up each URL, retrieve a release package, and locate the release inside it. In order to locate the specific release inside a release package, the `id` of the release must be appended to the package URL using a fragment identifier.
+For each `url` value, it must be possible for a consuming application to retrieve the release package at the URL and identify the release within it. Since a release package can contain multiple releases, for a linked release to identify a specific release via its `url` field, the `id` of the release must be appended to the release package URL using a fragment identifier.
 
-An example releases section is given below.
+The following example demonstrates the use of linked releases.
 
 ```eval_rst
 .. jsoninclude:: ../examples/merging/versioned.json
    :jsonpointer: /records/0
    :expand: releases, tag
    :title: releases
+```
+
+Above, the first linked release has a `url` value of <https://standard.open-contracting.org/examples/releases/ocds-213czf-000-00002-01-award1.json#ocds-213czf-000-00002-01-award1>. The first part (`https://standard.open-contracting.org/examples/releases/ocds-213czf-000-00002-01-award1.json`) is the URL of the release package, and the fragment identifier (`ocds-213czf-000-00002-01-award1`) is the `id` of the release.
+
+Release `id` values are only required to be unique within the scope of a contracting process: that is, within the scope of an `ocid` value. As such, a consuming application needs to use that fragment identifier in combination with the `ocid` of the record in order to identify the matching release within the release package.
+
+#### Embedded releases
+
+An embedded release follows the [release schema](reference).
```

Comment:
@jpmckinney maybe add "So instead of just linking the release, here the whole releases are embedded" or similar, to describe more what an embedded release is

Reply:
Good point. I'll use:

> In other words, instead of linking to a release within a release package as above, that release is entirely included in the record.